packages/e2e-tests/puppeteer/helpers.ts

```diff
@@ -217,8 +217,9 @@ export async function waitForBudgetEntry(page: Page): Promise<void> {
   await page.waitForSelector('.domain-budget-table > tbody > tr');
 }
 
+// For local runs, the default timeout of 30 seconds is not sufficient for a test with a 13MB file exchange.
```

Comment:
Why 13MB?

Reply:
That's how large our current `web3torrent-tests-stub` test file is. This comment can probably be better worded...

Reply:
Oh, I shrunk it in #1877

Reply:
Thanks for the heads up! 38092c16e